integration-tests/src/test/java/org/openmrs/maven/plugins/packager/config/ValidateConfigurationsTest.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.maven.plugins.packager.config;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openmrs.module.initializer.validator.Validator;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(Validator.class)
+public class ValidateConfigurationsTest {
+
+	@Before
+	public void before() {
+		PowerMockito.mockStatic(Validator.class);
+	}
+	
+	@Ignore
+	@Test
+	public void testConfigurationsValidated() throws Exception {
+		ConfigProject configProject = new ConfigProject("config-test-parent");
+		configProject.executeGoal("clean", "-N", "-X");
+		configProject.executeGoal("validate", "-N", "-X");
+		
+		PowerMockito.verifyStatic(Validator.class, times(1));
+		Validator.getJUnitResult(any(String[].class)); // TODO: make a better assert
+	}
```

Comment:
@mseaton this the main reason this PR is still WIP is because I couldn't get this to work...
The instruction
```java
configProject.executeGoal("validate", "-N", "-X");
```
doesn't actually get to `ValidateConfigurationsMojo#execute`, any idea why? I'm not too sure how this Maven `Verifier` works.

Reply:
@mks-d , it's possible I'm not doing things exactly right, but what I've found is that you need to add the executions you want to run explicitly into the pom.  So, here you are trying to use the "config-test-parent" pom to test the project.  That pom is found in src/test/resources/config-test-parent/pom.xml.  You'll need to add an execution to the end of that, something like this:

```                    
<execution>
  <id>validate-configurations</id>
  <phase>validate</phase>
    <goals>
    <goal>validate-configurations</goal>
  </goals>
</execution>
```